README.md

```diff
@@ -6,60 +6,114 @@
+* Events (`events/`)
+* Meetings & Meetups (`meetings-meetups/`) (note that these pages aren't linked from anywhere other than feeds, they are more for the metadata)
+* Projects (`projects/`)
+* Talks (`talks/`)
+
+Please see [this runbook page](http://runbook.ritlug.com/infrastructure/website/) for more info.
 
 ## Updating meeting times / places
 
 Edit `_config.yml` and update the settings "ritlug-time", "ritlug-place", and
-"ritlug-day". These are used (and should be used) everywhere on the website
+"ritlug-day".
+These are used (and should be used) everywhere on the website
 the meeting time and place are referenced so that they can be updated in
 one place.
-
+Please note that this does NOT update events in the iCal feed.
+
+## Site Source
+
+_(Basic Git & Ruby configs excluded)_
+
+* `_data/` - `.yml` files containing links on navbar tabs & in the footer
+* `_includes/`
+  * `content-blocks/` - reusable content blocks that can be inserted into a page with `{% include ... %}`
+  * `layout/` - common template chunks
+* `_layouts/` - page layouts. Which is used is defined in a given page's YAML frontmatter
+* `announcements/`, `events/`, `projects/`, `talks/` - See [Adding Content](#adding-content)
+* `css/styles.css` - Custom CSS
+* `feeds/` - iCal & Atom/RSS feeds
+* `img/` - images used across the entire site
+* `spec/` - code tests
+* `CNAME` - custom domain for GitHub Pages
+* `_config.yml` - Jekyll configuration
+* `about.md` - About page
+* `get-involved.md` - Getting Involved page
+* `index.html` - Home page
 
 ## Patches
 
-### Header Height
+### CSS
 
-The default header in Material Design Lite is too large on desktop so we've
-shrunk it. If you want to change the header height, do the following:
+Some changes can be found in `css/styles.css`.
 
-1. Open `css/styles.css`
-2. Search for `.mdl-demo .mdl-layout__header-row {`.
-3. Find and change the height. Default is 64px. We're currently using 25px.
+This includes:
+* Some margin/padding tweaks
+* RIT Orange
+* Use Roboto font for body & Overpass for headings (via Google Fonts CDN)
+* Footer tweaks
+  * Sticky footer (if page is less than the screen height, keep it at the bottom)
+  * Style lists to appear without bullets
 
 
 ## Developing and testing
 
-### Setup
-
-First, install Ruby:
+### Local Install
 
-```sh
-sudo (apt-get install|pacman -S|...) ruby
-```
+Please see the [Jekyll Installation Docs](https://jekyllrb.com/docs/installation/).
 
-Then, install bundler using gem.
+The server can be started with:
 
 ```sh
-sudo gem install bundler
+bundle exec jekyll serve
 ```
 
-Lastly, install dependencies with bundler.
+Which will serve the site at: http://localhost:4000
 
-```sh
-bundle install
-```
+**IF YOU HAVE ERRORS, PLEASE CHECK THAT YOU INSTALLED ALL REQUIRED DEPENDENCIES.**
+
+### Docker
 
-### Start the development server
+Jekyll can be run using [these Docker containers](https://github.com/envygeeks/jekyll-docker/blob/master/README.md).
```

Comment:
In the future we might want to consider adding @jwflory's docker script he used for the FOSS website in this repository.

Reply:
@jrtechs see L110-116

Reply:
Ah, thanks! I totally didn't see that :+1:

Reply:
It might be helpful to have a script to do all of the heavy-lifting for you, so a new contributor or developer can run the script without having to figure it all out.

Reply:
@jwflory I'm agnostic to whether they use a package or container as long as it runs. Your script is at L110-116

Reply:
@ct-martin I understand what you mean. Some people will not have a preference for which way to run it – they will prefer what works with the least trouble. Offering a script or another way of doing the tedious work for them could be helpful to add in a future PR.

Reply:
@jwflory as I said, your script is at lines 110-116 if they want to use it